model_card_template.md

```diff
@@ -0,0 +1,84 @@
+---
+language:
+- en
+library_name: transformers
+inference: false
+thumbnail: https://h2o.ai/etc.clientlibs/h2o/clientlibs/clientlib-site/resources/images/favicon.ico
+tags:
+- gpt
+- llm
+- large language model
+---
+# Model Card
+## Summary
+
+This model was trained using [H2O LLM Studio](https://github.com/h2oai/h2o-llmstudio).
+- Base model: [{{base_model}}](https://huggingface.co/{{base_model}})
+
+
+## Usage
+
+To use the model with the `transformers` library on a machine with GPUs, first make sure you have the `transformers` and `accelerate` libraries installed.
+
+```bash
+pip install transformers==4.28.1
+pip install accelerate==0.18.0
+```
+
+```python
+import torch
+from transformers import pipeline
+
+generate_text = pipeline(model="{{repo_id}}", torch_dtype=torch.bfloat16, trust_remote_code=True, device_map="auto")
+
+res = generate_text("Why is drinking water so healthy?", max_new_tokens=100)
```

Comment:
You may want to add special tokens in the example prompt, see also https://github.com/h2oai/h2o-llmstudio/issues/61

Reply:
What do you mean?

Reply:
But it will be more difficult for the chained prompts. Maybe, I need to rework it again.

Reply:
But it will be more difficult for the chained prompts. Maybe, I need to rework it again.

Reply:
I think we can ignore chained here, this should be rather clear.

Reply:
shall we potentially add the inference parameters from the experiment there?

Reply:
good idea, will add this

Reply:
`What do you mean? EOS, prompt start, answer start? These are added in the pipeline.`
Ok, I missed that